src/locales/ko/mystery-encounters/fight-or-flight-dialogue.json

```
@@ -1,25 +1,26 @@
 {
-  "intro": "Something shiny is sparkling\non the ground near that Pokémon!",
-  "title": "Fight or Flight",
-  "description": "It looks like there's a strong Pokémon guarding an item. Battling is the straightforward approach, but it looks strong. Perhaps you could steal the item, if you have the right Pokémon for the job.",
-  "query": "What will you do?",
+  "intro": "빛나는 무언가가 포켓몬 근처에서 반짝반짝 빛나고 있다!",
+  "title": "도망치거나, 맞서 싸우거나",
+  "description": "강한 포켓몬이 아이템을 지키고 있는 것 같습니다.\n그냥 싸워볼 수도 있지만, 조금 강해보이네요.\n적합한 기술을 가진 포켓몬이 있다면,\n아이템을 빼앗을 수도 있을 것 같습니다.",
+  "query": "어떻게 하시겠습니까?",
   "option": {
     "1": {
-      "label": "Battle the Pokémon",
-      "tooltip": "(-) Hard Battle\n(+) New Item",
-      "selected": "You approach the\nPokémon without fear.",
-      "stat_boost": "The {{enemyPokemon}}'s latent strength boosted one of its stats!"
+      "label": "포켓몬과 싸운다",
+      "tooltip": "(-) 어려운 배틀\n(+) 새 아이템",
+      "selected": "용감하게 포켓몬에게 다가갔다.",
+      "stat_boost": "{{enemyPokemon}}의 잠재력이 능력치 중 하나를 강화했다!"
     },
     "2": {
-      "label": "Steal the Item",
-      "disabled_tooltip": "Your Pokémon need to know certain moves to choose this",
-      "tooltip": "(+) {{option2PrimaryName}} uses {{option2PrimaryMove}}",
-      "selected": ".@d{32}.@d{32}.@d{32}$Your {{option2PrimaryName}} helps you out and uses {{option2PrimaryMove}}!$You nabbed the item!"
+      "label": "아이템을 훔친다",
+      "disabled_tooltip": "이 선택지를 선택하려면 포켓몬이 특정 기술을 가지고 있어야 합니다.",
+      "tooltip": "(+) {{option2PrimaryName}}의 {{option2PrimaryMove}} 사용",
+      "selected": ".@d{32}.@d{32}.@d{32}${{option2PrimaryName}}[[가]] {{option2PrimaryMove}}[[를]] 사용했다!$아이템을 훔쳤다!"
     },
     "3": {
-      "label": "Leave",
-      "tooltip": "(-) No Rewards",
-      "selected": "You leave the strong Pokémon\nwith its prize and continue on."
+      "label": "떠난다",
+      "tooltip": "(-) 보상 없음",
+      "selected": "강한 포켓몬과 상품을 남겨둔 채\n당신은 계속 나아간다."
```

```suggestion
      "selected": "강한 포켓몬과 전리품을 남겨둔 채\n당신은 계속 나아간다."
```
"상품"에는 사고파는 물건이라는 의미가 있어서 다른 단어로 바꾸는 게 나을 듯합니다. 일단 "전리품" 정도로 해 보았는데, 문제는 이 상황에서는 빼앗지 않은 물건을 칭하는 것이라 더 나은 단어가 있을 것 같기는 합니다...

---

일단은 이걸로